.gitignore

```diff
@@ -2,3 +2,4 @@ export
 .cache
 last_imported
 config.yml
+config-*.yml
```

Comment:
I'm wondering, maybe it would be nicer to introduce a `config` directory if there are multiple config files.

Maybe what could happen is:

* the script looks for a config.yml by default and if it doesn't find any `./config.yml` and no arguments are given, it throws an error explaining the two ways of configuring it
* when an argument is given, it never checks for the `./config.yml`, it could directly try to access `config/your-argument.yml`

What do you think?

I really like that idea, amazing that you started this PR!

Reply:
Cool. Will have to look how to do this with Ruby. I am more a Python/Bash guy ;)